spaces/domains.json

```
@@ -1,6 +1,7 @@
 {
   "vote.anarchydao.io": "anarchytoken.eth",
   "gov.tigris.trade": "tigrisdao.eth",
+  "gov.sapavault.xyz": "projectsapa.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "gov.sapavault.xyz": "projectsapa.eth",
```